doc/adoc/user_guide.adoc

```diff
@@ -157,11 +157,19 @@ manually. Once done, the upgrade process uses `zypper dup` and expects
 all required repositories to be setup correctly.
 
 Migration product target::
-If set, the migration will use that product as a target.
+The default product target is SLES-15-SP1. The migration_product setting can be used
+to change the product target. 
+
+[NOTE]
+Changing the product target is not officially supported
```

Comment:
This is somewhat misleading. While a user doesn't have to set it anymore we did have a bug a while back where the product was not detected and users had to use `migration_product`. Also if the users uses

`migration_product: 'SLE_SAP/15.1/x86_64'`

that is certainly supported. Of course there is not really a need for a user to do this anymore. However this in conjunction with line 160 gives the impression that `SLE_SAP` and `SLE_HPC` are not supported as products for migration. That is incorrect.

Reply:
Should we give the three examples, SLES,SLE_SAP and SLE_HPC and then point to  `/etc/products.d/baseproduct` ?  Or just point to  `/etc/products.d/baseproduct`  and make the example

migration_product: name/version/arch

Currently it has 

migration_product:  'SLES/15/x86_64'

which does not seem useful